docs/api-docs/getting-started/webhooks/webhook-events.md

```diff
@@ -59,7 +39,289 @@
+| store/cart/converted | This hook fires when a cart is converted into an order, which is typically after the payment step of checkout on the storefront. At this point, the cart is no longer accessible and has been deleted. This hook returns both the cart ID and order ID for correlation purposes. |
+
+The same response is returned for the following events.
+
+* `store/cart/created`
+* `store/cart/updated`
+* `store/cart/deleted`
+* `store/cart/abandoned`
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+
+### Note
+> The `store/cart/abandoned` hook is triggered independently of the Abandoned Cart Saver feature, which is only available on [select plans](https://www.bigcommerce.com/essentials/pricing/). To learn more about the Abandoned Cart Saver, see [Using the Abandoned Cart Saver](https://support.bigcommerce.com/s/article/Using-the-Abandoned-Cart-Saver).
+
+</div>
+</div>
+</div>
+
+**Response fields**
+- type -- can be cart or cart_line_item
+- id -- ID of the cart
+```json
+{
+    "scope": "store/cart/created",
+    "store_id": "1025646",
+    "data": {
+        "type": "cart",
+        "id": "09346904-4175-44fd-be53-f7e598531b6c"
+    },
+    "hash": "352e4afc6dd3fc85ea26bfdf3f91852604d57528",
+    "created_at": 1561482670,
+    "producer": "stores/{store_hash}"
+}
+```
+### The same response is returned for the following endpoints:
+* `store/cart/couponApplied`
+
+**Response fields**
+- type -- can be cart or cart_line_item
+- id -- ID of the cart
+- couponId -- ID of the coupon
+
+```json
+{
+    "scope": "store/cart/couponApplied",
+    "store_id": "1025646",
+    "data": {
+        "type": "cart",
+        "id": "09346904-4175-44fd-be53-f7e598531b6c",
+        "couponId": 1
+    },
+    "hash": "4b7297d295141b660e8db5a0d99dfcdf459fe825",
+    "created_at": 1561482761,
+    "producer": "stores/{store_hash}"
+}
+```
+### The same response is returned for the following endpoints:
+* `store/cart/converted`
+
+**Response fields**
+- type -- Will always be cart
+- id - ID of the cart
+- orderId - ID of the order created
+```json
+{
+    "scope": "store/cart/converted",
+    "store_id": "1025646",
+    "data": {
+        "type": "cart",
+        "id": "d30016e2-23c0-4a90-884f-2e92ac135476",
+        "orderId": 252
+    },
+    "hash": "b86db7c77d7ef8f90d6a8aefa56de32ccd776923",
+    "created_at": 1561486893,
+    "producer": "stores/{store_hash}"
+}
+```
+
+## Cart line item
+
+| Name | Description |
+|-|-|
+| store/cart/lineItem/* | Subscribe to all cart line item events. This webhook will fire when a change occurs to line items in the cart. This can be when items are added to a cart, removed or updated.(Ex. change to quantity, product options or price). |

```

Comment:
```suggestion
| store/cart/lineItem/* | Subscribe to all cart line item events. This webhook will fire when a change occurs to line items in the cart. This can be when items are added to a cart, removed, or updated. For example, change to quantity, product options, or price. |

```